hedera-node/hedera-app/src/main/java/com/hedera/node/app/store/ReadableStoreFactory.java

```diff
@@ -104,7 +101,6 @@ private static Map<Class<?>, StoreEntry> createFactoryMap() {
         // Block Records
         newMap.put(
                 ReadableBlockRecordStore.class, new StoreEntry(BlockRecordService.NAME, ReadableBlockRecordStore::new));
-        newMap.put(ReadableNodeStore.class, new StoreEntry(AddressBookService.NAME, ReadableNodeStoreImpl::new));
```

Comment:
I don't think this is related. If you remove this, there are others. Can you put this back to see if tests pass.